assertj-core/src/main/java/org/assertj/core/presentation/UnicodeRepresentation.java

```diff
@@ -33,6 +33,7 @@ public class UnicodeRepresentation extends StandardRepresentation {
   public String toStringOf(Object object) {
     if (hasCustomFormatterFor(object)) return customFormat(object);
     if (object instanceof String) return toStringOf((String) object);
+    if (object instanceof CharSequence) return toStringOf((CharSequence) object);
     if (object instanceof Character) return toStringOf((Character) object);
```

Comment:
I think technically all these `instanceof` checks are redundant because they are equivalent to what the parent class `StandardRepresentation` would do, but for consistency I added the check for `CharSequence` here as well.